client/my-sites/site-settings/form-general.jsx

```diff
@@ -552,7 +552,7 @@ export class SiteSettingsFormGeneral extends Component {
 
 		return (
 			<FormFieldset>
-				<FormLabel htmlFor="blogtimezone">{ translate( 'Site Timezone' ) }</FormLabel>
+				<FormLabel htmlFor="blogtimezone">{ translate( 'Site timezone' ) }</FormLabel>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.